### ISSUE TITLE
How to reset the mock location 

### ISSUE THREAD

**Issue by jas4954** [amotzte/android-mock-location-for-development#6]
This is working .When i checked my location in Google Maps its showing fake location.Now i want to reset to my real location.How can i do it?

**Comment by joao-jeronimo** [comment]
Hello.
For me it's otherwise: when I try to set the mock location it lasts for just a few seconds.

**Comment by jas4954** [comment]
This works only  when developer option is enabled.

**Comment by joao-jeronimo** [comment]
No. It lasts just for 2 seconds even if the "developer option" is enabled.

**Comment by joao-jeronimo** [comment]
See my fork fixing this bug:
https://github.com/joao-jeronimo/android-mock-location-for-development
